scripts/setup_integration/aws_integration_env.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+# aws_integration_env.sh
+#
+# SUMMARY
+#
+#   Builds and pulls down the Vector AWS Integration test environment
+
+# Echo usage if something isn't right.
+usage() {
+    echo "Usage: $0 [-a Action to run {stop|start} ] [-t The container tool to use {docker|podman} ] [-e The container enclosure to use {pod|network} ]" 1>&2; exit 1;
+}
+
+while getopts a:t:e: flag
+do
+    case "${flag}" in
+        a) ACTION=${OPTARG};;
+        t) CONTAINER_TOOL=${OPTARG};;
+        e) CONTAINER_ENCLOSURE=${OPTARG};;
+        :)
+        echo "ERROR: Option -$OPTARG requires an argument"
+        usage
+        ;;
+        *)
+        echo "ERROR: Invalid option -$OPTARG"
+        usage
+        ;;
+    esac
+done
+shift $((OPTIND-1))
+# Check required switches exist
+if [ -z "${ACTION}" ] || [ -z "${CONTAINER_TOOL}" ] || [ -z "${CONTAINER_ENCLOSURE}" ]; then
+    usage
+fi
+
+ACTION="${ACTION:-"stop"}"
+CONTAINER_TOOL="${CONTAINER_TOOL:-"podman"}"
+CONTAINER_ENCLOSURE="${CONTAINER_ENCLOSURE:-"pod"}"
```

Comment:
I would take this a step further. Since these are all called by a wrapper script `scripts/setup_integration/setup_integration_env.sh`, make these fixed required parameters, which makes checking them (if that's even needed) a simple matter. As described in another comment, that third parameter may be redundant as well.
```suggestion
if [ $# -ne 3 ]
then
    echo "Usage: $0 {stop|start} {docker|podman} {pod|network}" 1>&2; exit 1;
    exit 1
fi
ACTION=$1
CONTAINER_TOOL=$2
CONTAINER_ENCLOSURE=$3
```